### DISCUSSION TITLE
posts/2022-12-27-devinterrupted

### DISCUSSION THREAD

**Discussion by giscus** [cmdcolin/cmdcolin.github.io#13] [General]
# posts/2022-12-27-devinterrupted

Blogging for the future

https://cmdcolin.github.io/posts/2022-12-27-devinterrupted

**Comment by gladiatr72** [comment]
I've been a reddit lurker for years but have recently been hitting it directly since it seems I end up there anyway. Thank you + com-padres for your effort.
